### DISCUSSION TITLE
Multiple Inverters

### DISCUSSION THREAD

**Discussion by asdecopes** [OpenDTU-App/opendtu-react-native#94] [General]
Hi,
I've just found this app and I'm testing it with two different Hoymiles Inverters (HM-800 and HMT-2250). 
Could be possible to see the output power from my two inverters separatly in the live data?

For example:
Total output: 1500W
HM-800: 400W
HMT-2250: 1100W

Something like that.

Danke!

**Comment by CommanderRedYT** [comment]
This is already planned / something I'm working on right now. I can't really test the 2nd inverter other than emulating it with a snapshot of the data of the first one.
